drltrace/scripts/gen_drlttrace_config.py

```diff
@@ -0,0 +1,379 @@
+"HENHMETAFILE":"HENHMETAFILE", "HFONT":"HFONT", "HGDIOBJ":"HGDIOBJ", "HGLOBAL":"HGLOBAL",
+"HHOOK":"HHOOK", "HICON":"HICON", "HINSTANCE":"HINSTANCE", "HKEY":"HKEY", "HKL":"HKL",
+"HLOCAL":"HLOCAL", "HMENU":"HMENU", "HMETAFILE":"HMETAFILE", "HMODULE":"HMODULE",
+"HMONITOR":"HMONITOR", "HPALETTE":"HPALETTE", "HPEN":"HPEN", "HRGN":"HRGN",
+"HRSRC":"HRSRC", "HSTR":"HSTR", "HTASK":"HTASK", "HWINSTA":"HWINSTA", "HWND":"HWND",
+"SC_HANDLE":"SC_HANDLE", "SERVICE_STATUS_HANDLE":"SERVICE_STATUS_HANDLE", "int":"int",
+"char": "char", "wchar_t":"wchar", "PSRWLOCK": "SRWLOCK*", "ULONG_PTR": "ULONG_PTR",
+"PSID":"SID*", "PSECURE_MEMORY_CACHE_CALLBACK":"SECURE_MEMORY_CACHE_CALLBACK*",
+"PVOID":"VOID*", "VOID":"VOID", "PVECTORED_EXCEPTION_HANDLER":"VECTORED_EXCEPTION_HANDLER*",
+"PULONG_PTR":"ULONG_PTR*", "PBOOL":"BOOL*", "LPDWORD":"DWORD*", "LPVOID":"VOID*",
+"LPOVERLAPPED_COMPLETION_ROUTINE":"OVERLAPPED_COMPLETION_ROUTINE*", "LPDCB":"DCB*",
+"LPCOMMTIMEOUTS":"COMMTIMEOUTS*", "PTP_CALLBACK_INSTANCE": "TP_CALLBACK_INSTANCE*",
+"LPOVERLAPPED":"OVERLAPPED*", "PTP_IO": "TP_IO*", "LPCOMSTAT":"COMSTAT*",
+"PTP_POOL": "TP_POOL*", "PTP_CLEANUP_GROUP": "TP_CLEANUP_GROUP*", "PTP_TIMER": "TP_TIMER*",
+"PTP_WAIT": "TP_WAIT*", "PTP_WORK":"TP_WORK*", "LPCOMMCONFIG": "COMMCONFIG*",
+"FILETIME":"FILETIME", "PCNZWCH": "wchar*", "PNZWCH":"wchar*", "PUNZWCH": "wchar*",
+"PCUNZWCH":"wchar*", "LPBOOL":"bool*", "LPPROGRESS_ROUTINE":"PROGRESS_ROUTINE*",
+"INT":"int", "PCACTCTXA": "ACTCTXA*", "PCACTCTXW": "ACTCTXW*",
+"SECURITY_ATTRIBUTES": "SECURITY_ATTRIBUTES", "LPSECURITY_ATTRIBUTES": "SECURITY_ATTRIBUTES*",
+"SIZE_T": "size_t", "LPFIBER_START_ROUTINE": "FIBER_START_ROUTINE*",
+"PJOB_SET_ARRAY": "JOB_SET_ARRAY*", "PHANDLE": "HANDLE*",
+"MEMORY_RESOURCE_NOTIFICATION_TYPE": "MEMORY_RESOURCE_NOTIFICATION_TYPE",
+"LPTHREAD_START_ROUTINE": "THREAD_START_ROUTINE*", "PTP_WIN32_IO_CALLBACK": "TP_WIN32_IO_CALLBACK*",
+"PTP_CALLBACK_ENVIRON": "TP_CALLBACK_ENVIRON*", "PTP_TIMER_CALLBACK": "TP_TIMER_CALLBACK*",
+"PTP_WAIT_CALLBACK": "TP_WAIT_CALLBACK*", "PTP_WORK_CALLBACK": "TP_WORK_CALLBACK*",
+"WAITORTIMERCALLBACK": "WAITORTIMERCALLBACK", "LPPROC_THREAD_ATTRIBUTE_LIST": "PROC_THREAD_ATTRIBUTE_LIST*",
+"LPFILETIME": "FILETIME*", "DWORD64": "DWORD64", "CALINFO_ENUMPROCA": "CALINFO_ENUMPROCA",
+"CALINFO_ENUMPROCW": "CALINFO_ENUMPROCW", "CALID": "CALID", "CALINFO_ENUMPROCEXEX": "CALINFO_ENUMPROCEXEX",
+"CALINFO_ENUMPROCEXA": "CALINFO_ENUMPROCEXA", "CALINFO_ENUMPROCEXW": "CALINFO_ENUMPROCEXW",
+"CALTYPE": "CALTYPE", "DATEFMT_ENUMPROCA": "DATEFMT_ENUMPROCA", "DATEFMT_ENUMPROCEXA": "DATEFMT_ENUMPROCEXA",
+"DATEFMT_ENUMPROCW": "DATEFMT_ENUMPROCW", "DATEFMT_ENUMPROCEXW": "DATEFMT_ENUMPROCEXW",
+"DATEFMT_ENUMPROCEXEX": "DATEFMT_ENUMPROCEXEX", "LANGGROUPLOCALE_ENUMPROCA": "LANGGROUPLOCALE_ENUMPROCA",
+"LANGGROUPLOCALE_ENUMPROCW": "LANGGROUPLOCALE_ENUMPROCW", "LONG_PTR": "LONG_PTR",
+"ENUMRESLANGPROCA": "ENUMRESLANGPROCA", "ENUMRESLANGPROCW": "ENUMRESLANGPROCW", "ENUMRESNAMEPROCA": "ENUMRESNAMEPROCA",
```

Comment:
style: line too long.  See above: one per line will take more space but will be easier to read and maintain.